src/components/Modeler.vue

```diff
@@ -160,6 +158,7 @@ import registerInspectorExtension from '@/components/InspectorExtensionManager';
 
 import initAnchor from '@/mixins/linkManager.js';
 import { addIdToNodeAndSetUpDiagramReference, addNodeToProcess, getTargetProcess } from '@/components/nodeManager';
+import ensureShapeIsNotCovered from '@/components/shapeStackUtils';
```

Comment:
This name could be improved, but I figured it's better than `setShapeStacking`, and avoids describing what's happening in an imperative way: it should be clear that we're not concerned about setting specific z-indices, but about making sure a shape isn't covered or overlapping another shape.